src/map/lua/lua_baseentity.cpp

```diff
@@ -13576,6 +13577,38 @@ inline int32 CLuaBaseEntity::getTrickAttackChar(lua_State *L)
     return 1;
 }
 
+/************************************************************************
+*  Function: getTargetsWithinArea()
+*  Purpose :
+*  Example :
+*  Notes   :
+************************************************************************/
+
+inline int32 CLuaBaseEntity::getTargetsWithinArea(lua_State* L) {
+    TPZ_DEBUG_BREAK_IF(m_PBaseEntity->objtype == TYPE_NPC);
+
+    CBattleEntity* PEntity = (CBattleEntity*)m_PBaseEntity;
+
+    float radius = (float)lua_tonumber(L, 1);
+    uint8 allegiance = lua_isnil(L, 2) ? 0 : lua_tointeger(L, 2);
+
+    PEntity->PAI->TargetFind->reset();
+    PEntity->PAI->TargetFind->addAllInRange(PEntity, radius, allegiance);
+    uint16 size = (uint16)PEntity->PAI->TargetFind->m_targets.size();
+    lua_createtable(L, size, 0);
+	int i = 1;
```

Comment:
We have some leftover tabs floating around here (probably from my poor copy-pasting), please replace with 4xspaces

Reply:
Ive had a good look round, i cant see any more and i fixed this one, let me know if you see any more.